app/models/vacancy.rb

```diff
@@ -268,6 +268,16 @@ def delete_documents
     self.documents.each { |document| DocumentDelete.new(document).delete }
   end
 
+  def new_information
```

Comment:
How about calling this 'new_sections'? It reveals that it returns a list of sections and is more specific than 'information'

Reply:
This would change en.yml too